sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -490,7 +495,8 @@ public void setEnabled() throws InterruptedException {
         verify(channel).removeListener(isA(SessionTracker.class));
         verify(channel).removeListener(isA(AnalyticsValidator.class));
         verify(channel).removeListener(isA(AnalyticsTransmissionTarget.getChannelListener().getClass()));
-        verify(channel, times(2)).removeGroup(analytics.getGroupName());
+        verify(channel).removeGroup(eq(ANALYTICS_CRITICAL_GROUP));
+        verify(channel, times(2)).removeGroup(eq(ANALYTICS_GROUP));
```

Comment:
why is it asymmetric between the 2 groups? they should have the same life cycle, not sure to understand what is going on here

Reply:
Please ignore my comment I remember where this comes from, abstract code removes the group before adding it.